src/bindings/python/src/openvino/utils/types.py

```diff
@@ -161,15 +161,15 @@ def make_constant_node(value: NumericData, dtype: Union[NumericType, Type] = Non
     return const
 
 
-def as_node(input_value: NodeInput, name: Optional[str] = None) -> Union[Node, Output]:
+def as_node(input_value: NodeInput, name: Optional[str] = None) -> Union[Node, Output, Constant]:
     """Return input values as nodes. Scalars will be converted to Constant nodes."""
-    if issubclass(type(input_value), Node):
+    if isinstance(input_value, Node):
         return input_value
-    if issubclass(type(input_value), Output):
+    if isinstance(input_value, Output):
         return input_value
     return make_constant_node(input_value, name=name)
 
 
-def as_nodes(*input_values: NodeInput, name: Optional[str] = None) -> List[Node]:
+def as_nodes(*input_values: NodeInput, name: Optional[str] = None) -> List[Union[Node, Output]]:
```

Comment:
```suggestion
def as_nodes(*input_values: NodeInput, name: Optional[str] = None) -> List[Union[Node, Output, Constant]]:
```
because `as_node` returns `Union[Node, Output, Constant]` several lines above